clang/test/CIR/CodeGen/literals.c

```diff
@@ -0,0 +1,10 @@
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fclangir-enable -Wno-return-stack-address -emit-cir %s -o %t.cir
```

Comment:
Looks like `-Wno-return-stack-addres` is not needed, you can remove it. Also prefer not using `main` in the examples, pick any other function name unless there's anything special about `main` you are testing.